spring-boot-module/src/main/java/com/gmail/zagurskaya/online/cash/web/controller/AdministratorController.java

```diff
@@ -0,0 +1,166 @@
+package com.gmail.zagurskaya.online.cash.web.controller;
+
+import com.gmail.zagurskaya.online.cash.service.ReviewsService;
+import com.gmail.zagurskaya.online.cash.service.RoleService;
+import com.gmail.zagurskaya.online.cash.service.UserService;
+import com.gmail.zagurskaya.online.cash.service.model.ReviewsDTO;
+import com.gmail.zagurskaya.online.cash.service.model.RoleDTO;
+import com.gmail.zagurskaya.online.cash.service.model.UserDTO;
+import com.gmail.zagurskaya.online.cash.web.exception.NotFoundAllUsersException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class AdministratorController {
```

Comment:
need to separate functional from role. better to create functional controller - UserController, ReviewController etc